core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status=None, job_id=None):
```

Comment:
Generally, using default args leads to more mistakes since there's less auto-checking -- if there aren't any default args, you need to specify each arg exactly, and everything is very concrete.

The way you're proposing to use default args smells fishy, too. They shouldn't be used as signaling in that way, and that use case would suggest that the functions might be factored incorrectly. The intent (create/update) should determine which choice of function to call.